spec/support/fixtures.rb

```diff
@@ -114,3 +118,23 @@ class CustomParserModel < Base
     attribute  :age, parser: FixNumParser
   end
 end
+
+module Fixtures
+  class Association < PDC::Base
+    # stub the connection
+    SITE = 'https://pdc.host.dev.eng.pek2.redhat.com/'.freeze
+    self.connection = Faraday.new(url: SITE) do |faraday|
+      faraday.request   :json
+      faraday.use       PDC::Response::Parser
+      faraday.adapter   Faraday.default_adapter
+    end
+  end
+
+  class ProductVersion < Association
+    has_many :releases, uri: 'rest_api/v1/releases/?product_version=:product_version_id'
+  end
```

Comment:
When I test with Release class, I add this below line:
`      has_many :release_variants, uri: 'v1/release-variants/?release=:release_id'
`
It works, and there is no "rest_api" before v1.

Reply:
@xiangge Can u add more detail for this? You mean u add release_variants in a Release class and it doesn't need rest_api?